modules/gapi/samples/pipeline_modeling_tool/pipeline.hpp

```diff
@@ -192,26 +228,16 @@ class RegularPipeline : public Pipeline {
                             cv::GCompileArgs(m_args));
     }
 
-    virtual void init() override {
-        m_out_mats.resize(m_num_outputs);
-        for (auto& m : m_out_mats) {
-            m_pipeline_outputs += cv::gout(m);
-        }
-    }
-
-    virtual int64_t run_iter() override {
+    virtual double run_iter() override {
         using namespace std::chrono;
-        cv::gapi::wip::Data d;
-        m_src->pull(d);
-        auto in_mat = cv::util::get<cv::Mat>(d);
-        return utils::measure<milliseconds>([&]{
-            m_compiled(cv::gin(in_mat), cv::GRunArgsP{m_pipeline_outputs});
+        cv::gapi::wip::Data data;
+        m_src->pull(data);
+        return utils::measure<utils::double_ms_t>([&]{
```

Comment:
Does it work with `cv::MediaFrame` now?

Reply:
`wip::Data` is a wrapper over `GRunArg`. I guess it could work, but input is always `GMat` anyway.

Reply:
Thanks